doc: update docs for 2.6.2 (#17221)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/releases/release-notes-2.6.2.rst

Supported modem firmware
************************

See `Modem firmware compatibility matrix`_ for an overview of which modem firmware versions have been tested with this version of the |NCS|.

Use the latest version of the nRF Programmer app of `nRF Connect for Desktop`_ to update the modem firmware.
See :ref:`nrf9160_gs_updating_fw_modem` for instructions.

Modem-related libraries and versions
====================================

.. list-table:: Modem-related libraries and versions
   :widths: 15 10
   :header-rows: 1

   * - Library name
     - Version information
   * - Modem library
     - `Changelog <Modem library changelog for v2.6.2_>`_
   * - LwM2M carrier library
     - `Changelog <LwM2M carrier library changelog for v2.6.2_>`_

Known issues
************

Known issues are only tracked for the latest official release.
See `known issues for nRF Connect SDK v2.6.2`_ for the list of issues valid for the latest release.

.. _ncs_release_notes_262_changelog:

Changelog
*********

The following sections provide detailed lists of changes by component.

Protocols
=========

This section provides detailed lists of changes by :ref:`protocol <protocols>`.

Wi-Fi
-----

> * Fixed TX EVM regression.

Author: @krish2718 : Could you provide more details here?

> * Fixed TX EVM regression.

Reviewer: @srkanordic  can you please help here?

> * Fixed TX EVM regression.

Reviewer 2: This is a bug while configuring DPD coefficients. This is a bug in NCS release 2.6. This is causing TX EVM to degrade. This is fixed.

> * Fixed TX EVM regression.

Reviewer 3: could we rephrase like : _Fixed a bug that causes TX EVM to degrade while configuring DPD coefficients._ ?

> * Fixed TX EVM regression.

Reviewer 2: Do we need to explicitly say "while configuring DPD coefficients"?  If the details are not required then I would suggest to describe as follows - "Fixed a bug that causes TX EVM degradation"

See :ref:`nrf9160_gs_updating_fw_modem` for instructions.

Modem-related libraries and versions
====================================

.. list-table:: Modem-related libraries and versions
   :widths: 15 10
   :header-rows: 1

   * - Library name
     - Version information
   * - Modem library
     - `Changelog <Modem library changelog for v2.6.2_>`_
   * - LwM2M carrier library
     - `Changelog <LwM2M carrier library changelog for v2.6.2_>`_

Known issues
************

Known issues are only tracked for the latest official release.
See `known issues for nRF Connect SDK v2.6.2`_ for the list of issues valid for the latest release.

.. _ncs_release_notes_262_changelog:

Changelog
*********

The following sections provide detailed lists of changes by component.

Protocols
=========

This section provides detailed lists of changes by :ref:`protocol <protocols>`.

Wi-Fi
-----

* Fixed TX EVM regression.

Thread
------

* Fixed issue with high power consumption after parent loss.

Drivers
=======

This section provides detailed lists of changes by :ref:`driver <drivers>`.

Wi-Fi drivers
-------------

> * Added a new RPU recovery mechanism feature to recover the nRF70 Series chip.

Author: @krish2718 : Please confirm if this is correct.

Reviewer: Yes, this is correct, we could just do below
```suggestion
* Added a new recovery feature to recover the nRF70 Series chip in case of any lockups.
```

Author: Done.

Libraries
=========

This section provides detailed lists of changes by :ref:`library <libraries>`.

Libraries for networking
------------------------

* Added the :ref:`lib_wifi_ready` library.

Zephyr
======

> The Zephyr fork in |NCS| (``sdk-zephyr``) contains all commits from the upstream Zephyr repository up to and including ``23cf38934c0f68861e403b22bc3dd0ce6efbfa39``.

Author: @gmarull @shanthanordic: Could you please confirm the commit here?

Reviewer 4: @rlubos the last PR that we merged was noup right ? so it shouldnt it be  the cherry pick commit pointed by  https://github.com/nrfconnect/sdk-zephyr/commit/02ae5d8ded21e670b9930cb00fb64ae9310ed9dc ?

Reviewer 5: The commit used here (23cf38934c0f68861e403b22bc3dd0ce6efbfa39) looks correct to me, it's the merge base for the `v3.5.99-ncs1-branch` (I. e. the last commit from the upstream that came from a regular upmerge, which this line refers to).

Cherry-picks and noups are covered by the command in the line below.

Author: @rlubos : Please let me know if the commit or command below needs to be changed here.
